packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import i18n from 'i18next';
```

Comment:
When you implement the request change on providing a default i18n, you can remove those i18n props